util/storage-samples/storage-owner-sample/src/main/java/org/fido/iot/storage/OwnerDbManager.java

```diff
@@ -722,16 +782,6 @@ public String insertWgetContentHash(DataSource ds, String sviString) {
     return sb.toString();
```

Comment:
Failed to execute goal org.apache.maven.plugins:maven-checkstyle-plugin:3.1.0:check (validate) on project storage-owner-sample: Failed during checkstyle configuration

fido/iot/storage/OwnerDbManager.java:786:22: unexpected token: